code/obj/item/grab.dm

```diff
@@ -713,6 +713,9 @@
 		setProperty("I_disorient_resist", 15)
 
 	disposing()
+		for(var/datum/objectProperty/equipment/P in src.properties)
+			P.removeFromMob(src, src.assailant)
+			src.properties.Remove(P)
```

Comment:
I personally would leave this line out. I don't think `/obj` cleans up properties in disposing right now but it might do that in the future and I'd rather have all the properties there at that time.
```suggestion
```